horovod/run/run.py

```diff
@@ -559,6 +559,8 @@ def parse_args():
                                        help='Extra MPI arguments to pass to mpirun. '
                                        'They need to be passed with the equal sign to avoid parsing issues. '
                                        'e.g. --mpi-args="--map-by ppr:6:node"')
+    group_library_options.add_argument('--tcp', action='store_true', dest='tcp_flag',
+                                       help='If this flag is set, TCP is only used for communication.')
```

Comment:
It may be a knit-pick, but it might be clearer for the help text to read:
`'If this flag is set, only TCP is used for communication.'`